Classes/Service/TheodiaOrg.php

```diff
@@ -436,4 +447,21 @@ private function numberToRomanRepresentation(int $number): string
         }
         return $returnValue;
     }
+
+    /**
+     * Returns the HTTP header to sign the GraphQL query with the configured key
+     */
+    private function sign(Site $site, string $payload): string
+    {
+        $key = $site->getConfiguration()['tx_theodia_api_key'];
```

Comment:
Would be wise to add `?? null` to the end

Reply:
fixed